mozilla/foundation.mozilla.org
# Add a new image format "full width high quality"

Closes #6068

Some background research: https://github.com/mozilla/foundation.mozilla.org/issues/6068#issuecomment-765742148

@Pomax @KalobTaulien tagging you both for review as I'm not sure if this PR can cause any side effects. I've tested it locally on different pages and scenarios and they seemed fine. Also I'm wondering if we need to add this same file to all Wagtail apps we have? If so, is there a way to just have them referenced to the same one file instead of having duplicates?

---

Add a new image format "full width high quality" that is tailored for large high def image.
(Note: I can't re-order the format list without overwriting existing code, so I'm just gonne keep the new format appear last)

- Review app test page: https://foundation-s-issue-6068-goqawn.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/
- Edit page: https://foundation-s-issue-6068-goqawn.herokuapp.com/cms/pages/51/edit/



### Steps to test (review app or locally)
1. Go to any page that allows image embed in the rich text editor (e.g., Article page).
2. In rich text editor, embed the image Sabrina included in ticket #6068 (use Goole Drive one) and choose "full width high quality" as the format
3. Preview or publish the page
4. Inspect the image. See if it's 2280px wide instead of 800px.
5. You can also test on an existing image embed and verify the rendered image quality.
